csm/conf/etc/csm/csm.conf

```diff
@@ -128,6 +128,7 @@ CSM:
     username: "csm"
     password: ""
     password_decryption_key: "cortx"
+    session_backend: "in_memory"
```

Comment:
There should be two possible values for session_backend
1. persistent  -> for consul
2. local -> for in_memory